effekt/shared/src/main/scala/effekt/core/optimizer/StaticArguments.scala

```diff
@@ -120,22 +123,22 @@ object StaticArguments {
 
   def rewrite(s: Stmt)(using C: StaticArgumentsContext): Stmt = s match {
     case Stmt.Scope(definitions, body) =>
-      scope(rewrite(definitions), rewrite(body))
+      normal.Scope(rewrite(definitions), rewrite(body))
```

Comment:
This regularly is the only smart constructor we use since it is stupid to accidentally create a scope with no bindings.